app/src/stores/post.ts

```diff
@@ -42,7 +43,22 @@ export const usePostStore = defineStore("post", () => {
             return new Promise((resolve) => {
                 resolve(
                     posts.filter((post) => {
-                        return post.content[0]?.status == ContentStatus.Published;
+                        const isPublished = post.content[0]?.status == ContentStatus.Published;
+                        const publishDate = post.content[0]?.publishDate
+                            ? post.content[0]?.publishDate
+                            : null;
+                        const expiryDate = post.content[0]?.expiryDate
+                            ? post.content[0]?.expiryDate
+                            : null;
```

Comment:
Please don't use `null`: we use `undefined` consistently throughout the app. JavaScript is a bit problematic in that both `null` and `undefined` are a thing, in most languages it's just `null`. By using only one we do not run into any cases where we check something like `date == null` but if it's `undefined` then that will return `false` which is unexpected.

Reply:
Hi @laurensgroeneveld ! Thanks for the comment.  So instead of `null` i have to `indefined` right ?